src/plugins/roleColorEverywhere/index.tsx

```diff
@@ -126,11 +131,19 @@ export default definePlugin({
         {
             find: ".reactorDefault",
             replacement: {
-                match: /,onContextMenu:e=>.{0,15}\((\i),(\i),(\i)\).{0,250}tag:"strong"/,
+                match: /,onContextMenu:\i=>.{0,15}\((\i),(\i),(\i)\).{0,250}tag:"strong"/,
                 replace: "$&,style:{color:$self.getColor($2?.id,$1)}"
             },
             predicate: () => settings.store.reactorsList,
         },
+        {
+            find: ",reactionVoteCounts",
+            replacement: {
+                match: /\.nickname,(?=children:)/,
+                replace: "$&style:{color:$self.getColor(arguments[0]?.user?.id,{channelId:arguments[0]?.channel.id})},"
```

Comment:
```suggestion
                replace: "$&style:{color:$self.getColor(arguments[0]?.user?.id,{channelId:arguments[0]?.channel?.id})},"
```

or better yet, pass the raw arguments and handle it inside the function with try catch